### PR TITLE
Codespaces updates

### PR DESCRIPTION
**Pull request by sarajamous**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
-dwave-ocean-sdk==6.4.0
-matplotlib==3.3.4
+dwave-ocean-sdk>=6.4.0
+matplotlib~=3.3.4
```
Comment (randomir) [inline requirements.txt:2]: This limits `matplotlib` to `3.3.x`, i.e. only patch can vary. Quite restricted. 

For matplotlib, it's better to set lower bound only, as it's unlikely to break some basic functionality in the future.
```suggestion
matplotlib>=3.3.4
```

**Review by randomir — APPROVED** [review]

File: README.md
Comment (randomir) [inline README.md:1]: If you plan to run Jupyter in Codespaces, then I advise updating readme, like we did on dwave-examples, e.g. see [usage section here](https://github.com/dwave-examples/factoring-notebook/#usage).

Comment (sarajamous) [reply README.md:1]: Done, is it good to go now?

**Review by randomir — APPROVED** [review]
LGTM!

**Pull request merged**